vnext/Desktop.UnitTests/OriginPolicyHttpFilterTest.cpp

```diff
@@ -280,23 +280,20 @@ TEST_CLASS (OriginPolicyHttpFilterTest) {
     // Should implicitly set Content-Length and Content-Type
     request.Content(HttpStringContent{L"PreflightContent"});
 
-    auto filter = winrt::make<OriginPolicyHttpFilter>(mockFilter);
+    auto filter = winrt::make<OriginPolicyHttpFilter>("http://somehost", mockFilter);
     auto opFilter = filter.as<OriginPolicyHttpFilter>();
```

Comment:
That's not good to have this, someone can take over this Url.
Please use microsoft Urls at least, or ask Danny for best practice.

Reply:
These are unit tests.
No actual network call his being made to any of the indicated URLs.

They are used only for address-handling validations.